btree/btree.c

```diff
@@ -1139,54 +1298,84 @@ static struct nd *node_try(struct td *tree, struct segaddr *addr)
 {
 	return segops->so_node_try(tree, addr);
 }
+#endif
 
 static int64_t node_alloc(struct node_op *op, struct td *tree, int size,
-			  struct node_type *nt, struct m0_be_tx *tx, int nxt)
+			  const struct node_type *nt, int ksize, int vsize,
+			  struct m0_be_tx *tx, int nxt)
 {
-	return segops->so_node_alloc(op, tree, size, nt, tx, nxt);
+	int  nxt_state;
+	/**
```

Comment:
Doxygen does not work within function bodies.

Reply:
Yes, the point was that comments within a function body should start with "/* " rather than "/** ".